gems/activesupport/6.0/_test/test.rb

```diff
@@ -14,3 +14,5 @@
 nil.try(:tap) { |n| p n }
 nil.try { p 'hello' }
 nil.try { |n| p n }
+[1, 2, 3, 4].try { |n| p n.in_groups(2) }
```

Comment:
Why do you use `try`? This test is for `in_groups`. Do you have any reason to call `try` for testing `in_groups`?

Reply:
oops I followed the other tests. but It is certainly not needed and will be removed.